bin/copy-production-modules.js

```diff
@@ -0,0 +1,124 @@
+const fs = require( 'fs' ); // eslint-disable-line  import/no-nodejs-modules
```

Comment:
nit ` // eslint-disable-line  import/no-nodejs-modules` should be moved to `./bin/.eslintrc.js`. Importing node modules should be ok to any file in `./bin`

Reply:
Fixed :+1: